test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -641,7 +707,15 @@ void IHttpHeadersHandler.OnHeader(Span<byte> name, Span<byte> value)
                 ignoreNonGoAwayFrames: false,
                 expectedLastStreamId: 1,
                 expectedErrorCode: Http2ErrorCode.STREAM_CLOSED,
-                expectedErrorMessage: CoreStrings.FormatHttp2ErrorStreamClosed(Http2FrameType.DATA, streamId: 1));
+                expectedErrorMessage: null);
+
+            // There's a race where either of these messages could be logged, depending on if the stream cleanup has finished yet.
```

Comment:
Was this race there before or just after changing the scheduling mode? If it's the latter, revert this test change.